webdriver/webdriver.go

```diff
@@ -13,28 +15,71 @@ var (
 	startFrameBuffer = flag.Bool("frame_buffer", frameBufferDefault(), "Whether to use a frame buffer")
 	seleniumPath     = flag.String("selenium_path", "", "Path to the selenium standalone binary.")
 	seleniumHost     = flag.String("selenium_host", "localhost", "Host to run selenium on")
-	seleniumPort     = flag.Int("selenium_port", 8888, "Port to run selenium on")
+	seleniumPort     = flag.Int("selenium_port", 0, "Port to run selenium on")
```

Comment:
Will this yield picking a random port, or will it cause a failure when `selenium_port` is not specified?

Reply:
Reading further I see it's the former. Maybe add that to the doc string.